src/main/kotlin/com/bugsnag/android/gradle/SharedObjectMappingFileFactory.kt

```diff
@@ -152,21 +130,15 @@ internal object SharedObjectMappingFileFactory {
      *
      * @param stdout The input stream
      * @param outputFile The output file
-     * @param lineFilter A predicate which determines whether a line should be
      * included in the output file or not
      */
-    private fun outputZipFile(
-        stdout: InputStream,
-        outputFile: File,
-        lineFilter: (String) -> Boolean
-    ) {
+    private fun outputZipFile(stdout: InputStream, outputFile: File) {
         val src = stdout.source().buffer()
         val sink = outputFile.sink().gzip().buffer()
         src.use { source ->
             sink.use { gzipSink ->
                 // filter unnecessary lines from objdump here
                 generateSequence { source.readUtf8Line() }
-                    .filter(lineFilter)
                     .forEach { line -> gzipSink.writeString("$line\n", Charset.defaultCharset()) }
```

Comment:
Would [putting this back to using `writeAll`](https://github.com/bugsnag/bugsnag-android-gradle-plugin/blob/2e475db008a8f421b9f22f0807fa5786bd8313e3/src/main/kotlin/com/bugsnag/android/gradle/SharedObjectMappingFileFactory.kt#L138-L144) be more efficient than going line by line?

Reply:
Updated to use the old method.